.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,60 @@
+name: Bug report
+description: Report an issue
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Bug Description
+      description: Please describe the bug
+      value: "A clear and concise description of what the bug is"
+    validations:
+      required: true
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: Expected behavior
+      description: Please describe the behavior you expected
+      value: "A clear and concise description of what you expected to happen"
+    validations:
+      required: true
+  - type: textarea
+    id: experienced_behavior
+    attributes:
+      label: Experienced behavior
+      description: Please describe the behavior you experienced
+      value: "A clear and concise description of what actually happened"
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce the behavior
+      description: Please explain the steps to reproduce the experienced behavior
+      value: |-
+        Steps to reproduce the behavior
+        1.
+        2.
+        3.
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: Describe the environment you have
+      value: |-
+          - nri-plugin(s):
+          - container runtime:
+          - OS:
+          - kernel:
+          - Kubernetes version:
```

Comment:
can we add plugins version too in case there is some notable changes between versions which might cause different behaviors ?

Reply:
good point, added